### PR TITLE
Feature apps from database table

### PR DESCRIPTION
**Pull request by masterix21**
To create an app from command line:
`php artisan websockets:app:create`

To manage all existing apps, use the web panel from your browser:
`/laravel-websockets/admin`

### CODE REVIEW

**Comment by vesper8** [comment]
looks really useful! thanks for working on this!

I have two comments.

First, instanciate is mispelled, it should be instantiate

Second, in all of my websocket apps I use a conditional when setting the host, like this:

```
'host' => env('APP_ENV') === 'production' ? 'socket.mydomain.com' : 'socketmaster.dev',
```

Can your addition have any way of handling such a case ?

**Comment by masterix21** [comment]
@vesper8 thanks for the first comment.

About the second, i think that if you're not using the same database between production and local, the problem is not real.

**Comment by vesper8** [comment]
@masterix21 good point! didn't think of that

is it possible to enable/disable apps without deleting them?

**Comment by masterix21** [comment]
- Added enable/disable apps feature
- Added pagination to apps index
- Added search to apps index

**Comment by veneliniliev** [comment]
this is great feature... please merge it :)

**Comment by eldor** [comment]
Until this has been merged or if you want to do it in Nova you can use this package
[https://packagist.org/packages/vemcogroup/nova-websockets](https://packagist.org/packages/vemcogroup/nova-websockets)

**Comment by homeserverinc** [comment]
Sorry about my question, but, until this commits are merged and released with the official composer package, how could I use this great piece of software with my current composer installed package of laravel-websockets?

I mean, what is the more correct way to achieve this?

**Comment by masterix21** [comment]
You can use my fork in your composer.json (as local package), but without 5.8 support.

**Comment by homeserverinc** [comment]
Ok, thanks @masterix21. 

I will try that way.

File: src/Database/AppProvider.php [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    }
+
+    public function findById($appId): ?App
+    {
+        return $this->instantiate(DatabaseApp::find($appId)->toArray() ?? null);
+    }
+
+    public function findByKey(string $appKey): ?App
+    {
+        return $this->instantiate(DatabaseApp::where('key', $appKey)->first()->toArray() ?? null);
```
Comment (kpilard) [inline src/Database/AppProvider.php]: @masterix21 If you submit an unknown appKey to this function the websocket-service will crash.
in this cas `DatabaseApp::where('key', $appKey)->first()` is null and `null->toArray()` throws an exception

**Comment by rennokki** [comment]
@masterix21 Can you rebase?

**Comment by rennokki** [comment]
@masterix21 I assume you need to update your fork's master branch with the current one.

**Pull request closed without merging**